src/javascript/app_2/App/Components/Elements/InkBar/div_with_inkbar.jsx

```diff
@@ -0,0 +1,85 @@
+import classnames from 'classnames';
+import PropTypes  from 'prop-types';
+import React      from 'react';
+import { InkBar } from './inkbar.jsx';
+
+class DivWithInkBar extends React.Component {
```

Comment:
I don't really like having `With` in the name, can we remove it, or change it to InkBarDiv instead?